tests/example-czechitas.spec.ts

```diff
@@ -10,12 +10,16 @@ test.describe("Example test class for functionality showcase", () => {
     await pages.visitPage();
   });
 
-  test("contactsPageUrlTest", async () => {
+  test("When user navigates to Contacts section then user is navigated to czechitas url", async () => {
```

Comment:
user is not navigated to czechitas url, it just checks that link is there. 🤔 
Btw. I would delete this testcase due it doesn't make sense to me to check it. 
It checks, that `link with url www.czechitas.cz` has www.czechitas.cz text.. 🤦

Reply:
changing it to something more meaningful - like checking correct address is shown